ui/terminal/table.go

```diff
@@ -0,0 +1,47 @@
+package terminal
+
+import (
+	"os"
+	"strings"
+)
+
+type tablePrinter interface {
```

Comment:
Perhaps this could live in the terminal package, and define a statusPrinter rather than being table specific?